ToyBox/classes/MonkeyPatchin/BagOfPatches/LevelUpPatchesWrath.cs

```diff
@@ -487,12 +487,31 @@ private static class CharGenClassSelectorItemVM_SpecialRequiredRace_Patch {
             }
         }
 
+        [HarmonyPatch(typeof(BlueprintCharacterClass))]
+        private static class BlueprintCharacterClass_Patch {
+            [HarmonyPatch(nameof(BlueprintCharacterClass.MeetsPrerequisites)), HarmonyPostfix]
+            public static void MeetsPrerequisites_Patch(ref bool __result, BlueprintCharacterClass __instance) {
+                if (settings.toggleIgnoreFeaturePrerequisitesWhenChoosingClass &&
+                    !__instance.PrestigeClass && !__instance.IsMythic) {
```

Comment:
Any reason for excluding PrestigeClasses?

Reply:
Because the previous patch didn't seem to work on Prestige/Mythic, and I assumed that was intentional. Can remove it if you think it shouldn't be there

Reply:
Considering the name of the setting I'd consider it in scope; so probably easier to remove